include/zephyr/bluetooth/audio/has.h

```diff
@@ -213,6 +214,46 @@ int bt_has_client_conn_get(const struct bt_has *has, struct bt_conn **conn);
  */
 int bt_has_client_presets_read(struct bt_has *has, uint8_t index, uint8_t max_count);
 
+/**
+ * @brief Switch Active Preset.
+ *
+ * Client method to set preset identified by @p index as active.
+ * The status is returned in the @ref bt_has_client_cb.preset_switch callback.
+ *
+ * @param has Pointer to the Hearing Access Service object.
+ * @param index Preset index to activate.
+ * @param sync Request active preset synchronization in set.
+ *
+ * @return 0 in case of success or negative value in case of error.
+ */
+int bt_has_client_preset_switch(struct bt_has *has, uint8_t index, bool sync);
+
+/**
+ * @brief Activate Next Preset.
+ *
+ * Client method to set next available preset as active.
```

Comment:
Just to make sure I understand this, if the current preset is index 5, then this would set it the next preset (e.g. 6), right?

What happens if there are not a higher preset? Does it wrap around again to 1?

Reply:
Yes. This behavior is defined in spec

Reply:
@MariuszSkamra I think it would make sense to document the behavior in the API as well